src/logic/export/RectLabelsExporter.ts

```diff
@@ -54,6 +62,81 @@ export class RectLabelsExporter {
         return labelRectsString.join("\n");
     }
 
+    private static exportAsVOC(): void {
+        let zip = new JSZip();
+        store.getState().editor.imagesData.forEach((imageData: ImageData) => {
+            const fileContent: string = RectLabelsExporter.wrapImageIntoVOC(imageData);
+            if (fileContent) {
+                const fileName : string = imageData.fileData.name.replace(/\.[^/.]+$/, ".xml");
+                zip.file(fileName, fileContent);
+            }
+        });
+
+        const projectName: string = store.getState().editor.projectName
+            .toLowerCase()
+            .replace(' ', '-');
+
+        const date: string = moment().format('YYYYMMDDhhmmss');
+        zip.generateAsync({type:"blob"})
+            .then(function(content) {
+                saveAs(content, "labels_" + projectName + '_' + date + ".zip");
+            });
+    }
+
+    private static wrapRectLabelsIntoVOC(imageData: ImageData): string {
+        if (imageData.labelRects.length === 0 || !imageData.loadStatus)
+            return null;
+
+        const labelNamesList: string[] = store.getState().editor.labelNames;
+        const labelRectsString: string[] = imageData.labelRects.map((labelRect: LabelRect) => {
+            const labelFields = [
+                "\t<object>",
```

Comment:
hmm. jeśli dobrze rozumiem ten format, to on niczym (na tym etapie) się nie różni od XMLa... weź wiec może skorzystaj z jakiejś npm-owej paczki (np: https://www.npmjs.com/package/xml )